CONTRIBUTING.md

```diff
@@ -76,6 +76,17 @@ var/path_type = "/obj/item/weapon/baseball_bat"
 
 * You must use tabs to indent your code, NOT SPACES.
 
+* Control statements (such as if(), while(), spawn(), etc) must be followed by a new line. It is not allowed to follow a control statement with a statement in the same line.
```

Comment:
I recommend you offload this to @MrStonedOne 's PR.
he is already making significant changes to CONTRIBUTING.md
https://github.com/tgstation/-tg-station/pull/13196